src/khoj/interface/web/search.html

```diff
@@ -4,13 +4,13 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0 maximum-scale=1.0">
         <title>Khoj - Search</title>
 
-        <link rel="icon" type="image/png" sizes="128x128" href="/static/assets/icons/favicon-128x128.png">
-        <link rel="manifest" href="/static/khoj.webmanifest">
-        <link rel="stylesheet" href="/static/assets/khoj.css">
+        <link rel="icon" type="image/png" sizes="128x128" href="/static/assets/icons/favicon-128x128.png?v={{ khoj_version }}">
```

Comment:
Is the rationale behind including the `khoj_version` to limit the caching to the current version for static files? Neat solution.

Reply:
Yes, exactly!